CHANGELOG.rst

```diff
@@ -47,6 +47,8 @@ LMS: Support adding students to a cohort via the instructor dashboard. TNL-163
 
 LMS: Show cohorts on the new instructor dashboard. TNL-161
 
+LMS: Extended hints feature
```

Comment:
Yes, I'll put TODO there or something as an official reminder that a docs-handoff is needed at PR time. Ross has stuff for it.